js/loadTrack.js

```diff
@@ -1,24 +1,22 @@
 $('.trackLink').click(function(e) {
-    var link = e.target.dataset.track
-        
-    if  (link === '1' && $('#track1').length === 0) {
-        $('#container').append($('<div>').load('includes/trackLayout1.html'))
-    } else if (link === '1' && $('#track1').length !== 0) {
-        $('.track').css('display', 'none')
-        $('#track1').css('display', 'block')
-    }
+    var trackNo = e.target.getAttribute("data-track"),
+        trackElement = $('#track' + trackNo)
+
+    navigateLevels(trackElement, trackNo)
+})
 
-    if  (link === '2' && $('#track2').length === 0) {
-        $('#container').append($('<div>').load('includes/trackLayout2.html'))  
-    } else if (link === '2' && $('#track2').length !== 0) {
+function navigateLevels (trackElement, trackNo) {
+    if  (trackElement.length === 0) {
+        $('#container').append($('<div id="track' + trackNo + '" class="track">').load('includes/trackLayout' + trackNo + '.html'))
+    } else {
         $('.track').css('display', 'none')
-        $('#track2').css('display', 'block')
+        trackElement.css('display', 'block')
     }
 
-    $('#splash').css('display', 'none')  
-})
+    $('#splash').css('display', 'none')
+}
 
 $('body').on('click', '.trackReturn', function() {
```

Comment:
move to the top of the file with the other event listener for easy reading

Reply:
🎉